string code
geometry_msgs/Point left
geometry_msgs/Point right
geometry_msgs/Point top
geometry_msgs/Point bottom
geometry_msgs/Point matrix_center
float64 matrix_height
float64 matrix_width